source/components/menu/HxMenu.js

```diff
@@ -0,0 +1,117 @@
+import { getPosition } from '../../lib/position';
+
+window.addEventListener('WebComponentsReady', function () {
+    const tagName = 'hx-menu';
+
+    class HxMenu extends HTMLElement {
+        static get is () {
+            return tagName;
+        }
+
+        static get observedAttributes () {
+            return [ 'open' ];
+        }
+
+        constructor () {
+            super();
+            var menuId = this.getAttribute('id');
+            this.$menu = document.querySelector(`[aria-controls="${menuId}"]`);
+            this._onDocumentClick = this._onDocumentClick.bind(this);
+        }
+
+        connectedCallback () {
+            this._upgradeProperty('open', 'position');
+            document.addEventListener('click', this._onDocumentClick);
+        }
+
+        disconnectedCallback () {
+            document.removeEventListener('click', this._onDocumentClick);       
+        }
+
+        attributeChangedCallback (attr, oldValue, newValue) {
+            switch (attr) {
+                case 'open':
+                    this.setAttribute('aria-expanded', newValue !== '');
+                    break;
+            }
```

Comment:
Now that we're only listening for one attribute, we don't need the `switch`.

`this.setAttribute('aria-hidden', newValue !== '');`